### ISSUE TITLE
Specify minimal version required to use federated subscription with @apollo/subgraph

### ISSUE THREAD

**Issue by bnjjj** [apollographql/router#3870]
We should add a note somewhere in our documentation for people using apollo server that they have to upgrade their version because it's not available in previous versions. (I don't know which one exactly introduced the federated subscription support)

**Comment by JoeDevine** [comment]
Absolutely! and not just that `@apollo/server` needs updating, but also the specific version of `@apollo/subgraph` as well.
